models/divide_operation.rb

```
@@ -0,0 +1,9 @@
+require_relative 'operation'
+
+class DivideOperation < Operation
+  def operate(first_operand, second_operand)
+    raise ZeroDivisionError, 'Cannot divide by zero!' if second_operand.zero?
+
+    first_operand / second_operand
```

I'm pretty sure Ruby will raise this error. Could you explain what you need to check it?

---

From what I've read Ruby uses the IEEE 754 Floating Point Standard so without raising this error the result of the operation will be "Infinity", which is tecnically not an error value but would confuse the user.

---

ooooo nice catch!

---

Actually I need to raise the error or render the page again if the user tries dividing 0 / 0, it's currently returning NaN :no_mouth: 
Thanks for questioning this implementation, without it I wouldn't have caught that!